modules/packages/LinearAlgebra.chpl

```diff
@@ -1367,6 +1367,91 @@ proc cholesky(A: [] ?t, lower = true)
 }
 
 
+/* Find the eigenvalues of a real-symmetric/complex-hermitian matrix
+   ``A``. ``A`` must be square.
+
+   The algorithms uses either the lower-triangular (if ``lower`` is ``true``,
+   or upper-triangular part of the matrix only. If ``overwrite`` is
+   true, on exiting, this part
+   of the matrix, including the diagonal is overwritten.
+
+   .. note::
+
+     This procedure currently just returns all eigenvalues. Future
```

Comment:
> Future versions may allow the user to selectively return certain eigenvalues.

I would prefer to track this feature request in an issue rather than the documentation. Maybe we should remove this sentence from the docs, leaving:

```
This procedure currently just returns all eigenvalues.
To return selectively return certain eigenvalues, the user should 
call the LAPACK routine directly.
```